components/coreapi/blocks.go

```diff
@@ -0,0 +1,67 @@
+package coreapi
+
+import (
+	"github.com/labstack/echo/v4"
+	"github.com/pkg/errors"
+
+	"github.com/iotaledger/inx-app/pkg/httpserver"
+	"github.com/iotaledger/iota-core/pkg/restapi"
+	iotago "github.com/iotaledger/iota.go/v4"
+)
+
+func blockBytesByID(c echo.Context) ([]byte, error) {
```

Comment:
there's a lot of duplicate code here. I think all 3 methods can be combined to always parse the blockIDParam and get it from the engine and afterwards extract the necessary information / format